skel/lib/systemd/system-generators/dcache-generator

```diff
@@ -35,4 +42,6 @@ for domain in $(getProperty dcache.domains); do
 	[Install]
```

Comment:
Can you please remove the `[Install]` section at the same time? Since you cannot 'enable' a generated unit, there is no point in having this section. The symlink you add below is the substitute for this.

Reply:
Uhm... at least the Debian postgresql guys have it in their instance unit, too (even though ours isn't a true instance unit).

Further, this should go in a separate commit, I think... i.e. one that cleans up the generated units... these commits are just about adding the global one.

Reply:
The postgresql guys do not generate a dynamic unit - they just add the symlinks. It does make sense to have it in `postgresql@.service` and in `dcache@.service` since these are not dynamically generated and thus can be 'enabled'.

Leave it in if you like - it will not allow you to enable the generated units though, but It will not cause harm either.

Reply:
You are right.

I leave it for now, and let's clean it up once we know where we'll eventually go with the whole thing.